soc/arm64/intel_socfpga/agilex/mmu_regions.c

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2021, Intel Corporation. All rights reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <soc.h>
+#include <arch/arm64/arm_mmu.h>
+
+static const struct arm_mmu_region mmu_regions[] = {
+
+	MMU_REGION_FLAT_ENTRY("PERIPHERAL",
+			      0xF7000000,
+			      0x08E00000,
+			      MT_DEVICE_nGnRnE | MT_RW | MT_SECURE),
```

Comment:
Use `MT_P_RW_U_RW` or similar instead of `MT_RW`

Reply:
Noted.